websites/C/Carl-bot/presence.ts

```diff
@@ -1,16 +1,14 @@
 const presence = new Presence({
   clientId: "653372675166568481"
-});
-
-const browsingStamp = Math.floor(Date.now() / 1000);
+}),
+browsingStamp = Math.floor(Date.now() / 1000);
 
 presence.on("UpdateData", () => {
   const presenceData: PresenceData = {
     largeImageKey: "carllogo",
     startTimestamp: browsingStamp
   };
 
-  if (document.location.hostname == "carl.gg") {
```

Comment:
by this you are removing check and this second check to check hostname will not work lol, restore it and fix eslint issues

Reply:
In my testing it appeared to work just fine but I'll edit it regardless.